cfme/tests/automate/test_method.py

```diff
@@ -348,3 +350,99 @@ def test_send_email_method(smtp_test, klass):
     # TODO(GH-8820): This issue should be fixed to check mails sent to person in 'cc' and 'bcc'
     # Check whether the mail sent via automate method really arrives
     wait_for(lambda: len(smtp_test.get_emails(to_address=mail_to)) > 0, num_sec=60, delay=10)
+
+
+@pytest.fixture(scope="module")
+def generic_definition(appliance):
+    # Creating generic object using REST
+    with appliance.context.use(ViaREST):
+        definition = appliance.collections.generic_object_definitions.create(
+            name="LoadBalancer_{}".format(fauxfactory.gen_numeric_string(3)),
+            description="LoadBalancer",
+            attributes={"location": "string"},
+            associations={"vms": "Vm", "services": "Service"}
+        )
+        yield definition
+        if definition.exists:
+            definition.delete()
+
+
+@pytest.fixture
+def go_service_request(request, generic_catalog_item):
+    # Creating generic service
+    service_request = generic_catalog_item.appliance.rest_api.collections.service_templates.get(
+        name=generic_catalog_item.name
+    ).action.order()
+    request.addfinalizer(service_request.action.delete)
```

Comment:
I prefer using `yield` in fixtures like this one, that way there's no need to pass `request` to the fixture. But I'll leave that as optional.